src/validations/members.js

```diff
@@ -0,0 +1,31 @@
+const Joi = require('joi');
+
+const validateMemberUpdate = (req, res, next) => {
+  const memberValidation = Joi.object({
+    firstName: Joi.string().min(2).max(30),
+    lastName: Joi.string().min(2).max(30),
+    dni: Joi.string().min(8).max(12).integer(),
+    phone: Joi.string().min(6).max(20).integer(),
```

Comment:
integer is not parameter from a string, if you change data type, you can validate with length()

Reply:
integer removed